modules/channel-web/src/messaging-migration/down-sqlite.ts

```diff
@@ -177,4 +193,16 @@ export class MessagingSqliteDownMigrator extends MessagingDownMigrator {
       this.messageBatch = []
     }
   }
+
+  private async emptySessionBatch() {
+    if (this.sessionBatch.length > 0) {
+      for (const session of this.sessionBatch) {
+        await this.trx('dialog_sessions')
+          .update({ id: session.newId })
+          .where({ id: session.oldId })
+      }
+
+      this.sessionBatch = []
+    }
+  }
```

Comment:
Is it really necessary to put this in a batch if were doing individual updates?

Reply:
I totally agree. I added it for consistency's sake, but it could indeed be done directly instead of by using batches. I personally don't think that it would change things much, so I will keep it this way.